wsapi/wsapiStructs.go

```diff
@@ -267,6 +267,16 @@ type AddressRequest struct {
 
 type HeightRequest struct {
 	Height int64 `json:"height"`
+	NoRaw  bool  `json:"noraw,omitempty"`
+}
+
+func (hr *HeightRequest) IncludeRaw() bool {
+	return !hr.NoRaw
+}
+
+type ReplayRequest struct {
+	StartHeight uint32 `json:"startheight"`
+	EndHeight   uint32 `json:"endheight,omitempty"`
 }
```

Comment:
What is this struct doing in this PR? Is that a glitch of the rebase?

Reply:
Apparently so. My apologies. It's removed.